Client/Helpers/FxALoginHelper.swift

```diff
@@ -55,6 +55,11 @@ class FxALoginHelper {
             return loginDidFail()
         }
 
+        // If we've got here then we're loading an already logged in account.
+        // At this point, no one is interested if the account is verified or not,
+        // so assume it is.
+        accountVerified = true
```

Comment:
We can't assume that because we're able to get a handle to the account that it's verified. You might be able to use the `actionNeeded` property to make sure we're not in a pre-verified state. It's kind of annoying though because we're conflating the `.none` state with being verified which might not always be true.